packages/web-components/fast-element/src/platform.ts

```diff
@@ -79,6 +104,56 @@ if ($global.trustedTypes === void 0) {
     $global.trustedTypes = { createPolicy: (n: string, r: TrustedTypesPolicy) => r };
 }
 
+const propConfig = {
+    configurable: false,
+    enumerable: false,
+    writable: false,
+};
+
+if ($global.FAST === void 0) {
+    Reflect.defineProperty($global, "FAST", {
+        value: Object.create(null),
+        ...propConfig,
+    });
+}
+
+if ($global.FAST.getById === void 0) {
+    const storage = Object.create(null);
+
+    Reflect.defineProperty($global.FAST, "getById", {
+        value<T>(id: string | number, initialize?: () => T): T | null {
+            let found = storage[id];
+
+            if (found === void 0) {
+                found = initialize ? (storage[id] = initialize()) : null;
+            }
+
+            return found;
+        },
+        ...propConfig,
+    });
+}
+
+// TODO: Replace with ??= after TS 4.x update
```

Comment:
Can we put the issue https://github.com/microsoft/fast/issues/5198 in here for easier searchability?